build.ts

```diff
@@ -144,33 +342,44 @@ async function makeThemeObject(
 
 async function generateVariants(
   name: string,
-  palette: Labels<Color, AlphaColor>
+  flavor: CatppuccinFlavor,
 ) {
   for (const accent of accents) {
-    const theme = await makeThemeObject(name, accent, palette);
+    const theme = await makeThemeObject(name, accent, flavor);
+    const darkLightTheme = await makeDarkLightThemeObject(name, accent, flavor);
+    const fileName = `${name}-${accent}.xpi`;
+    writeTheme(fileName, theme, `./themes/${name}`);
+    writeTheme(fileName, darkLightTheme, `./themes/dark-light/${name}`);
+  }
+}
+
+async function writeTheme(
+  fileName: string,
+  theme: unknown,
+  path: string,
+) {
+  if (theme) {
     const json = JSON.stringify(theme, undefined, 2);
 
-    Deno.mkdirSync(`./themes/${name}`, {
+    Deno.mkdirSync(`${path}`, {
       recursive: true,
     });
 
     const zip = new JSZip();
     zip.addFile("manifest.json", json);
 
-    const images = zip.folder("images")
-    images.addFile("icon16.png", "./assets/icon16.png")
-    images.addFile("icon48.png", "./assets/icon48.png")
-    images.addFile("icon128.png", "./assets/icon128.png")
+    const images = zip.folder("images");
+    images.addFile("icon16.png", "./assets/icon16.png");
+    images.addFile("icon48.png", "./assets/icon48.png");
+    images.addFile("icon128.png", "./assets/icon128.png");
 
-    await zip.writeZip(`./themes/${name}/${name}-${accent}.xpi`);
+    await zip.writeZip(`${path}/${fileName}`);
   }
 }
 
 const start = performance.now();
 await Promise.all(
-  Object.entries(variants).map(([name, palette]) =>
-    generateVariants(name, palette)
-  )
+  flavorEntries.map(([name, flavor]) => generateVariants(name, flavor)),
```

Comment:
```suggestion
  flavorEntries.map(([identifier, flavor]) => generateVariants(identifier, flavor)),
```
Catppuccin's palette metadata has distinct differences between "identifiers" and "names" (e.g. `frappe` is the identifier whereas `Frappé` is the name), it helps to keep it consistent in variable naming as well.

Reply:
This one I still could do. I guess the Name could be used in the title and the identifier everywhere else.

Reply:
done :)